core/src/main/java/org/p2p/core/utils/Constants.kt

```diff
@@ -24,6 +24,7 @@ object Constants {
     const val USDT_COINGECKO_ID = "tether"
 
     const val WRAPPED_SOL_MINT = "So11111111111111111111111111111111111111112"
+    const val FEE_RELAYER_ACCOUNT = "FG4Y3yX4AAchp1HvNZ7LfzFTewF2f6nDoMDCohTFrdpT"
```

Comment:
Don't we have some value in the response to determine that payer is fee relayer account?
It may be changed and you'll have incorrect fees

Reply:
we do not
and i thought that it is our feeRelayer address and it is a constant